src/main/kotlin/com/lsd/core/adapter/puml/SequenceDiagramMarkup.kt

```diff
@@ -69,7 +69,7 @@ private fun Message.arrowMarkup(): String {
 }
 
 fun Participant.toParticipantMarkup(): String =
-    "${type.name.lowercase()} ${componentName.name}${alias?.let { " as \"$alias\"" } ?: ""}"
+    "${type.name.lowercase()} ${componentName.name}${alias?.let { " as \"$alias\"" } ?: ""}${colour?.let { " $colour" } ?: ""}"
```

Comment:
Hey, thinking we could add the `#` here in the markup i.e. `" #$colour"` so that it's not needed when creating a `Particpant` i.e.  `ACTOR.called("Arnie", null, "green");`

Reply:
Done